server/src/models/song.model.ts

```diff
@@ -0,0 +1,23 @@
+import { Schema, model, models } from 'mongoose';
+import { SongSchema } from '../types/song.types';
+
+const songSchema = new Schema<SongSchema>({
+  title: { type: String, required: true },
+  tempo: [{ type: String, required: true }],
+  originalKey: { type: String, required: true },
+  tags: [{ type: String, required: true }],
+  artist: { type: String, required: true },
+  firstLine: { type: String, required: true },
+  id: { type: Schema.Types.ObjectId },
```

Comment:
you dont need to define `id` because `_id` is autogenerated by mongo. please delete